src/test/java/no/unit/nva/handlers/UserDtoCreator.java

```diff
@@ -0,0 +1,101 @@
+package no.unit.nva.handlers;
+
+import static nva.commons.utils.JsonUtils.objectMapper;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.io.InputStream;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Collections;
+import no.unit.nva.exceptions.InvalidRoleInternalException;
+import no.unit.nva.exceptions.InvalidUserInternalException;
+import no.unit.nva.model.RoleDto;
+import no.unit.nva.model.UserDto;
+import no.unit.nva.testutils.HandlerRequestBuilder;
+
+public interface UserDtoCreator {
```

Comment:
It is used only in tests for creating users of different configurations

Reply:
I know it works, but I don't see why this needs to be an interface with default methods.